terraform/helm/testnet-addons/values.yaml

```diff
@@ -9,6 +9,11 @@ genesis:
   # prefix used to get the genesis configuration
   username_prefix: aptos-node
 
+waypoint:
```

Comment:
if we set a resource limit for the waypoint container it should always try to allocate on bigger machine right?

Reply:
yea that's true, assuming that the resources are available within the schedule-able nodegroups. our `utility` nodegroup is generally memory optimized. we should have both resource limit and affinity/taint tbh, but for now this serves our needs